trajectories:
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 0.
    magnitude_pitch: 0.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 45.
    magnitude_pitch: 0.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 90.
    magnitude_pitch: 0.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 135.
    magnitude_pitch: 0.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 180.
    magnitude_pitch: 0.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: -45.
    magnitude_pitch: 0.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: -90.
    magnitude_pitch: 0.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: -135.
    magnitude_pitch: 0.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)

  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 0.
    magnitude_pitch: 15.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 45.
    magnitude_pitch: 15.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 90.
    magnitude_pitch: 15.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 135.
    magnitude_pitch: 15.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 180.
    magnitude_pitch: 15.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: -45.
    magnitude_pitch: 15.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: -90.
    magnitude_pitch: 15.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: -135.
    magnitude_pitch: 15.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)

  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 0.
    magnitude_pitch: -15.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 45.
    magnitude_pitch: -15.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 90.
    magnitude_pitch: -15.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 135.
    magnitude_pitch: -15.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 180.
    magnitude_pitch: -15.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: -45.
    magnitude_pitch: -15.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: -90.
    magnitude_pitch: -15.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: -135.
    magnitude_pitch: -15.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)

  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 0.
    magnitude_pitch: 30.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 45.
    magnitude_pitch: 30.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 90.
    magnitude_pitch: 30.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 135.
    magnitude_pitch: 30.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 180.
    magnitude_pitch: 30.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: -45.
    magnitude_pitch: 30.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: -90.
    magnitude_pitch: 30.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: -135.
    magnitude_pitch: 30.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)

  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 0.
    magnitude_pitch: -30.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 45.
    magnitude_pitch: -30.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 90.
    magnitude_pitch: -30.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 135.
    magnitude_pitch: -30.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: 180.
    magnitude_pitch: -30.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: -45.
    magnitude_pitch: -30.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: -90.
    magnitude_pitch: -30.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)
  - type: acceleration
    frame: $(param tf_prefix)/look_ahead_point_stabilized
    magnitude: $(param magnitude)
    magnitude_yaw: -135.
    magnitude_pitch: -30.
    dt: $(param dt)
    ht: $(param ht)
    max_velocity: $(param max_velocity)